samples/nrf_desktop/CMakeLists.txt

```diff
@@ -79,6 +88,14 @@ if (${CONFIG_BOOTLOADER_MCUBOOT})
     message(FATAL_ERROR "Path to MCUBoot must be specified")
```

Comment:
we could also set `MCUBOOT_DIR` automatically here since we have west and multirepo
set(BOARD_ROOT ${CMAKE_CURRENT_SOURCE_DIR}/../../../mcuboot)

Reply:
ok